frontend/src/metabase/query_builder/components/SavedQuestionHeaderButton/SavedQuestionHeaderButton.styled.jsx

```diff
@@ -1,17 +1,23 @@
 import styled from "@emotion/styled";
-import { color } from "metabase/lib/colors";
 
-import Button from "metabase/core/components/Button";
+import EditableText from "../EditableText/EditableText";
+import Icon from "metabase/components/Icon";
 
-export const HeaderButton = styled(Button)`
-  font-size: 1.25rem;
-  border: none;
-  padding: 0.25rem 0.25rem;
-  color: ${props => (props.isActive ? color("brand") : "unset")};
-  background-color: ${props => (props.isActive ? color("bg-light") : "unset")};
-  text-align: left;
+export const Root = styled.div`
+  ${EditableText.Root}::after, ${EditableText.TextArea} {
```

Comment:
Could we remove `data-replicated-value` at all? You can use regular props for pseudo elements like `:after` too https://github.com/metabase/metabase/blob/fc479abc46ef97158a65a01f601513001ac49267/frontend/src/metabase/query_builder/components/EditableText/EditableText.styled.tsx#L21